Website/web/src/components/NavBar.jsx

```diff
@@ -16,6 +16,7 @@ const NavBar = () => {
           <img alt="logo" src={templogo} style={{ width: "55px" }} />
         </div>
         <div className="nav-links">
+          <Link to="/">Home</Link>
```

Comment:
While you're in this file maybe add some icons to each link to make things look nicer. And maybe highlight the current link being viewed. Like if the user is on the dashboard page, highlight the "Dashboard" in the nav bar